lib/Sema/Constraint.cpp

```diff
@@ -50,9 +50,9 @@ Constraint::Constraint(ConstraintKind kind, ArrayRef<Constraint *> constraints,
 Constraint::Constraint(ConstraintKind Kind, Type First, Type Second,
                        ConstraintLocator *locator,
                        SmallPtrSetImpl<TypeVariableType *> &typeVars)
-    : Kind(Kind), HasRestriction(false), IsActive(false), IsDisabled(false),
-      IsDisabledForPerformance(false), RememberChoice(false), IsFavored(false),
-      IsIsolated(false),
+    : Kind(Kind), HasFix(false), HasRestriction(false), IsActive(false),
+      IsDisabled(false), IsDisabledForPerformance(false), RememberChoice(false),
+      IsFavored(false), IsIsolated(false),
```

Comment:
I’d appreciate if we could land some of these after favoring changes since it takes time to rebase all these small changes in a proper way because favoring changes modify places in CSStep and Constraint…

Reply:
Ok sure

Reply:
Actually I checked and this PR does not cause any conflicts with solver-perf, however there is a trivial conflict currently because of this that already landed on main: https://github.com/swiftlang/swift/pull/77152/commits/e0b6a96f0e9fae3aa02128263b89356c8d09b62d